lib/cracking.rb

```diff
@@ -0,0 +1,38 @@
+require_relative './cryptor'
+require_relative './decryptor'
+
+
+class Cracking < Cryptor
```

Comment:
changed name of class from `Crack` to` Cracking` to allow for a runner file named `crack.rb`